repair/row_level.cc

```diff
@@ -3009,17 +3008,13 @@ future<> repair_service::load_history() {
             auto repair_time = to_gc_clock(entry.ts);
             rlogger.debug("Loading repair history for keyspace={}, table={}, table_uuid={}, repair_time={}, range={}",
                     entry.ks, entry.cf, entry.table_uuid, entry.ts, range);
-            co_await get_db().invoke_on_all([entry, range, repair_time] (replica::database& local_db) -> future<> {
+            co_await get_db().invoke_on_all([&entry, range, repair_time] (replica::database& local_db) {
```

Comment:
nitpick: it might be a little more idiomatic to just capture the needed value here instead of a reference: `table_uuid = entry.table_uuid`

Reply:
yes, it's possible

Reply:
(in the next patch... this one still needs the reference to entry for printing the warning)